cheroot/test/helper.py

```diff
@@ -40,7 +41,7 @@ class CherootWebCase(webtest.WebCase):
     scheme = 'http'
 
     available_servers = {
-        'wsgi': cheroot.server.WSGIServer,
+        'wsgi': cheroot.wsgi.WSGIServer,
```

Comment:
Yeah, I also feel weird for what I just did.

 I start to get familiar with GitHub quite recently. Thank you for all these hints. ^o^